controllers/profiles/reconciler_dev.go

```diff
@@ -280,55 +317,63 @@ func getDeploymentFailureMessage(deployment *appsv1.Deployment) string {
 }
 
 // mountDevConfigMapsMutateVisitor mounts the required configMaps in the Workflow Dev Deployment
-func mountDevConfigMapsMutateVisitor(flowDefCM, propsCM *v1.ConfigMap) mutateVisitor {
+func mountDevConfigMapsMutateVisitor(flowDefCM, propsCM *v1.ConfigMap, externalResourceConfigMaps map[string]v1.ConfigMap) mutateVisitor {
 	return func(object client.Object) controllerutil.MutateFn {
 		return func() error {
```

Comment:
this func returns a error but there is nothing in this func that can return it, maybe remove or return it?

Reply:
@spolti  Is a design choice of @ricardozanini , let wait to know what is his thought

Reply:
the only thing returned is nil, maybe you can return the errors from the functions called inside this func.